libsession/src/main/java/org/session/libsession/utilities/ListUtil.kt

```diff
@@ -0,0 +1,28 @@
+package org.session.libsession.utilities
+
+/**
+ * @return the last element that satisfies the [predicate].
+ *
+ * This function has undefined behavior if:
```

Comment:
Why 'undefined behaviour'?
Isn't the behaviour what you described above? Even if multiple are true, the last one will be returned, won't it?

Reply:
Or do you mean:
```
 * This function has undefined behavior if:
 * - The given [predicate] will return [true] for some number of elements, and the elements are not sorted 
 *   according to the predicate. 
 ```